addons/core/package.json

```diff
@@ -45,44 +45,47 @@
     "intl-unofficial-duration-unit-format": "^3.1.0"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.18.2",
-    "@babel/plugin-proposal-decorators": "^7.18.6",
+    "@babel/eslint-parser": "^7.21.3",
+    "@babel/plugin-proposal-decorators": "^7.21.0",
     "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.8.1",
+    "@ember/string": "^3.0.1",
+    "@ember/test-helpers": "^2.9.3",
     "@embroider/test-setup": "^2.1.1",
     "@faker-js/faker": "^8.0.2",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
     "broccoli-asset-rev": "^3.0.0",
+    "concurrently": "^8.0.1",
     "doctoc": "^2.2.0",
-    "ember-cli": "^4.4.1",
+    "ember-cli": "^4.12.2",
     "ember-cli-dependency-checker": "^3.3.1",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
-    "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-export-application-global": "^2.0.1",
     "ember-inline-svg": "^1.0.1",
     "ember-load-initializers": "^2.1.2",
     "ember-page-title": "^7.0.0",
-    "ember-qunit": "^6.0.0",
-    "ember-resolver": "^8.0.3",
-    "ember-source": "~4.4.0",
-    "ember-template-lint": "^4.14.0",
+    "ember-qunit": "^6.2.0",
+    "ember-resolver": "^10.0.0",
+    "ember-source": "~4.12.0",
+    "ember-source-channel-url": "^3.0.0",
+    "ember-template-lint": "^4.18.2",
```

Comment:
@calcaide after some testing `ember-template-lint: "^4.18.2"` and `prettier: "^2.7.1"` are our allowed upgrades for our template linting.  Pushing to > `ember-template-lint: "5.0.0"` has some changes that I will add to tech time to discuss.

Reply:
Thanks for the heads up!!

Reply:
Can you give a quick explanation what the issues are?

Reply:
@ZedLi I don't think its really an issue its just an idea of how we use quotes and also different indentation in our test files.  I am still trying to compare the releases and find out what changed from `ember-template-lint: "4.18.2"` to `ember-template-lint: "5.7.2"` which is what ember addons are using. Currently its only targeting the `tests` files.  So we can either let the linter do its work or we can use overrides to cancel linting on test.  Thoughts

Reply:
I assume the issue is that we're now linting the HBS code that were embedded which was [included](https://github.com/ember-template-lint/ember-template-lint/releases/tag/v5.0.0) in v5? And that's why it's only in tests?

The quotes should always be single quotes though

Reply:
@ZedLi yes that is correct. Ok I will push it back to 5.72 and put a message out in boundary-ui channel